DQM/GEM/interface/GEMDQMBase.h

```diff
@@ -28,7 +28,7 @@
 
 class GEMDQMBase : public DQMEDAnalyzer {
 public:
-  // Borrwed from DQMOffline/Muon/interface/GEMOfflineDQMBase.h
+  // Borrwed from DQM/GEM/interface/GEMOfflineDQMBase.h
```

Comment:
Please revert (ok, it is a comment line, but let have consistent comments in the release)